FaunaDB.Client/Client/DefaultClientIO.cs

```diff
@@ -29,9 +32,12 @@ internal DefaultClientIO(HttpClient client, AuthenticationHeaderValue authHeader
             this.lastSeen = lastSeen;
         }
 
-        public DefaultClientIO(string secret, Uri endpoint, TimeSpan timeout)
-            : this(CreateClient(endpoint, timeout), AuthHeader(secret), new LastSeen())
-        { }
+        public DefaultClientIO(string secret, Uri endpoint, TimeSpan timeout, HttpClient httpClient = null)
+            : this(CreateClient(endpoint, timeout, httpClient), AuthHeader(secret), new LastSeen())
```

Comment:
no need to pass `httpClient` to the function `CreateClient`, if you already have it, just ignore the function call.
```csharp
httpClient ?? CreateClient(endpoint, timeout)
```

Reply:
I need to pass the httpClient if the user passes their own in FaunaClient(httpClient: userDefinedClient)

Reply:
wait I see what you are saying on this one

Reply:
Ya we can ignore the call too I guess